src/ontology/components/mondo-subsets.owl

```diff
@@ -189501,5 +189498,5 @@
 
 
 
-<!-- Generated by the OWL API (version 4.5.25) https://github.com/owlcs/owlapi -->
+<!-- Generated by the OWL API (version 4.5.25.2023-02-15T19:15:49Z) https://github.com/owlcs/owlapi -->
```

Comment:
can you confirm that you intended to make these changes and this is okay?

Reply:
I don't know... :-) 

@matentzn I obsoleted a term with "nord-rare", and removed the "nord-rare" subset annotation for that obsoleted term. I assume this is why this change is showing and that it is ok. BUT I don't know for sure. 
Could you please confirm? Thank you!

Reply:
No, you can't change that file manually! No rare subset should be edited manually by us, even during obsoletion!

Reply:
It wasn't changed manually, it was changed automatically by changing things in the ontology (so I guess it is semi-manual?).
We, therefore, need to answer the following question: What do we do when obsoleting a term that is in the rare disease subset (whether it is because of merge or obsoletion alone)? Do we leave the rare subset annotations on the obsoleted term? (and wait for the release to clean it up)?
(is this going to be a problem with QC?)

Reply:
> Do we leave the rare subset annotations on the obsoleted term? (and wait for the release to clean it up)?

Not only do we leave them on the obsoleted terms - for the rare subset, we _keep_ them there. We want to reflect the subset with 100% fidelity to what we have been provided with - if this means that an obsoleted term is used, this should be reflected. Only the rare disease resources should be able to fix this, and we need to establish a line of communications with them to enable them how to do so.